src/stream.js

```diff
@@ -21,8 +26,63 @@ module.exports.handler = async (event) => {
 	delete(groupedRecords['deleteMe']); // delete bad events
 
 	for (const records of Object.entries(groupedRecords)) {
-		const latestRecord = records.pop();
+		logger.debug({records}, 'Records');
+		const latestRecord = records.pop().pop();
+		// Tag for a transient record that should be ignored.
+		if (latestRecord.date === '_current') {
```

Comment:
Another approach would be to add this check to the filtering on line 15 so we could guarantee in this loop that we have no `_current` records.

Reply:
Line 15 filters non-parsed events.

Reply:
Yes but I was suggesting adding another filter condition: discard DELETE events _and_ records where `date === _current`.